.claude/skills/worktree/SKILL.md

```
@@ -242,10 +247,77 @@ Directory suffix is auto-derived from the branch name:
+   ```bash
+   wt.exe -w 0 new-tab --title "<title>" -d "<windows-path>"
+   ```
+
+   - `-w 0` targets the current Windows Terminal window (adds a tab, does not open a new window)
+   - `-d <path>` sets the tab's starting directory -- wt honors the backslash Windows-style path
+   - No trailing command. The default profile starts naturally.
+   - Keep calls sequential (one Bash call per worktree). Each call returns exit 0 immediately; wt fires-and-forgets.
+
+3. **Report:**
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)